angular/test/protractor/group_spec.coffee

```diff
@@ -182,3 +280,25 @@ describe 'Group Page', ->
                  '.change-volume-form__submit')
 
       page.expectText('.group-volume-card', 'Email everything')
+
+  describe 'starting a discussion', ->
```

Comment:
Great to have some robust e2ez around this stuff... will be really nice to feel like the new group form is really sorted.